pkgs/by-name/pi/picom-pijulius/package.nix

```diff
@@ -0,0 +1,19 @@
+{ picom, lib, fetchFromGitHub }:
+picom.overrideAttrs (oldAttrs: rec {
+  pname = "picom-pijulius";
+  version = "e7b1488";
+
+  src = fetchFromGitHub {
+    owner = "pijulius";
+    repo = "picom";
+    rev = "${version}";
+    sha256 = "1xi7bk8yicdvpn17cjws9q4xff8dcgl664i6pkdyhfwwg3j6j1b1";
+    fetchSubmodules = true;
+  };
+
+  meta = with lib; {
+    description = "Pijuliu's picom fork";
+    homepage = "https://github.com/pijulius/picom";
+    maintainers = with maintainers; oldAttrs.meta.maintainers ++ [ YvesStraten ];
```

Comment:
You should not add the maintainers from the base package with out asking them if that is okey with them.

Reply:
Sorry, I just did what Picom Jonaburg did as I thought its right, will remove